### PR TITLE
Add KuzzleRequest.getSearchBody documentation

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do ?

Add documentation about `KuzzleRequest.getSearchBody` method

### Other changes

  - fix  `cluster:health` api action documentation

### CODE REVIEW

**Review by jenow — APPROVED** [review]

File: doc/2/framework/classes/kuzzle-request/get-search-body/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description: KuzzleRequest class getSearchBody() method
+---
+
+# getSearchBody
+
+<SinceBadge version="2.11.0" />
+
+Returns the search body.
+
+With the HTTP protocol, the search body will be fetch depending on the verb:
```
Comment (xbill82) [inline doc/2/framework/classes/kuzzle-request/get-search-body/index.md:14]:
```suggestion
In the HTTP protocol, the search body will be extracted depending on the verb:
```

File: doc/2/framework/classes/kuzzle-request/get-search-body/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - `GET`: extract the `searchBody` query string argument and parse it
+  - `POST`: return the request body
+
+Typically the search body contains keywords used by Elasticsearch to perform queries like `query`, `sort`, `aggregations` etc.
```
Comment (xbill82) [inline doc/2/framework/classes/kuzzle-request/get-search-body/index.md:18]: I don't understand the purpose of this sentence.

**Review by xbill82 — CHANGES_REQUESTED** [review]

File: doc/2/framework/classes/kuzzle-request/get-search-body/index.md (hunk 2 above)
Comment (Aschen) [reply doc/2/framework/classes/kuzzle-request/get-search-body/index.md:18]: People are often confused about what is inside the search body (one of our client for example)

File: doc/2/framework/classes/kuzzle-request/get-search-body/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+<SinceBadge version="2.11.0" />
+
+Returns the search body.
+
+In the HTTP protocol, the search body will be extracted depending on the verb:
+  - `GET`: extract the `searchBody` query string argument and parse it
+  - `POST`: return the request body
+
+Typically the search body contains keywords used by Elasticsearch to perform queries like `query`, `sort`, `aggregations` etc.
```
Comment (Aschen) [inline doc/2/framework/classes/kuzzle-request/get-search-body/index.md:18]:
```suggestion
Typically the search body contains keywords used by Kuzzle to perform queries like `query`, `sort`, `aggregations` etc.
```

**Review by xbill82 — APPROVED** [review]

**Review by Leodau — CHANGES_REQUESTED** [review]

**Review by Shiranuit — APPROVED** [review]

**Pull request merged**